### PR TITLE
#861 Highlight current Text box

### LINKED ISSUE #861: Highlight current Text box
Image

The Select box has a red underline when selected, do the same for the Text box and other controls.

Note, this isn't currently done else where.

The colour might be changed from orange to grey though (orange is to close to red -> error)

### PR DESCRIPTION
**Pull request by roxy-dao**
Closes #861 

Might want to sneak this into develop branch as well... or update program with develop and vice versa~

- Added red underline when text + select components are in focus
- Also changed the currency input to have red underline on focus or grey underline on hover

### CODE REVIEW

File: client/packages/common/src/ui/components/inputs/CurrencyInput/CurrencyInput.tsx [hunk 1]
```diff
@@ -22,6 +22,12 @@ const StyledCurrencyInput = styled(RCInput)(({ theme }) => ({
   border: 'none',
   '&:focus': {
     outline: 'none',
+    borderBottom: '2px solid #FF0000',
+    borderRadius: '8px 8px 0px 0px',
+  },
+  '&:hover': {
+    borderBottom: '2px solid #4F4F4F',
+    borderRadius: '8px 8px 0px 0px',
```
Comment (clemens-msupply) [inline client/packages/common/src/ui/components/inputs/CurrencyInput/CurrencyInput.tsx:30]: The red line is gone now when the mouse is over the control, this is different to the other controls. Could you please have a look?

**Review by clemens-msupply — COMMENTED** [review]
Thanks, yes maybe just change this PR to develop? (@mark-prins)

When there is an error it looks a bit funny (double red line). But maybe this is for another issue (if you don't have a good idea how to make it look nicer)?

image

**Comment by roxy-dao** [comment]
> Thanks, yes maybe just change this PR to develop? (@mark-prins)
> 
> When there is an error it looks a bit funny (double red line). But maybe this is for another issue (if you don't have a good idea how to make it look nicer)?
> 
> image

The other red line will be for error message 🤔  Maybe we should change the colour on focus?

File: client/packages/common/src/ui/components/inputs/CurrencyInput/CurrencyInput.tsx (hunk 1 above)
Comment (roxy-dao) [reply client/packages/common/src/ui/components/inputs/CurrencyInput/CurrencyInput.tsx:30]: Sorry, I'm not sure what you mean. Which control is this happening on? 

This section is for the currency input
Screen Shot 2022-11-30 at 2 19 58 PM
Red line for focus, and when you hover over it's a grey line like the other input/select components

**Comment by mark-prins** [comment]
the suggestion was to use secondary as the focus colour - because our primary colour looks too much like an error.
I would prefer the blue for the focus - I had a comment in one of the issues which did this, just set `color` to `secondary` I think.

And yes - can you target `develop` and we'll merge develop into the programs feature 🙏

**Comment by roxy-dao** [comment]
> 061e7e2

Changed the focus line to blue and disabled underline when there's an error, so that it doesn't double up with error line (:

**Comment by roxy-dao** [comment]
Re-opening with new branch from develop~

**Pull request closed without merging**